cmd/migration-manager/internal/cmds/target.go

```diff
@@ -260,7 +260,7 @@ func (c *cmdTargetList) Run(cmd *cobra.Command, args []string) error {
 				authType = "TLS"
 			}
 
-			data = append(data, []string{t.Name, incusProperties.Endpoint, authType, strconv.FormatBool(incusProperties.Insecure)})
+			data = append(data, []string{t.Name, "Incus", incusProperties.Endpoint, authType, strconv.FormatBool(incusProperties.Insecure)})
```

Comment:
Any reason not to get that from t.TargetType instead of the hardcoded string?

Reply:
I suppose not, other than we're in a `case` where we know we're dealing with an Incus target already.

Reply:
Yeah, I'm just thinking that down the line it will be cleaner if we can add the row outside of the switch statement and just have the switch statement fetch the additional data for each of the types, so one less hardcoded thing for when we do that ;)